templates/todo/common/infra/bicep/app/apim-api-policy.xml

```diff
@@ -0,0 +1,91 @@
+<!-- Policy configuration for the API. Explore other sample policies at https://learn.microsoft.com/en-us/azure/api-management/policies/ -->
```

Comment:
With this are we now saying that this file has to be included in every template?  What are some ways we can avoid having to do that?

Reply:
We could put the content inside the apim-api.bicep file but that will be less elegant. It's a problem to include the file?

Reply:
Is there anything in here that is Todo specific?  Can it belong in core?

Reply:
It's not app specific. I will move it to core

Reply:
fixed